src/graph/graph.c

```diff
@@ -14,6 +14,9 @@
 #include "../util/rmalloc.h"
 #include "../util/datablock/oo_datablock.h"
 
+// Minimimum buffer size for matrices if enabled.
+#define MIN_MATRIX_BUFFER 100
```

Comment:
This is just here to avoid using hard-coded constants, as per Dvir's recommendation. It can replace the bool in the config, but I feel like that's a bit disingenuous as it's not actually user-configurable.